GenericBot/CommandModules/ModCommands.cs

```diff
@@ -572,6 +572,36 @@ public List<Command> GetModCommands()
             };
 
             ModCommands.Add(archive);
+            
+            Command replace = new Command("replace");
```

Comment:
Move all this into the >config command in ConfigCommands.cs instead of its own command